src/server/log.js

```diff
@@ -14,7 +14,6 @@ function error(error) {
     }
     console.log(chalk.red.bold('SIM ERROR: ' + error));
     config.server && config.server.close();
-    process.exit(1);
```

Comment:
I added [a function for that](https://github.com/Microsoft/cordova-simulate/pull/11/files/334e563898a5d13a539f575dcbebac6f4d6e711f#diff-6ddd72d4edcfc54d2078a3ddee13c0c7R55).